content/managed-addons/monitoring-stack/grafana-dashboard.md

```diff
@@ -12,139 +12,139 @@ This document explains how to create Grafana Dashboard via GrafanaDashboard CR p
 
    You can choose any existing namespace. The label `grafanaDashboard: grafana-operator` is required for Grafana Operator to discover the dashboard. The JSON string with the dashboard contents is placed in the `json` section. Check the [official documentation](https://grafana.com/docs/reference/dashboard/#dashboard-json) for details on JSON Model. You can create a dashboard via web UI first, and then export the dashboard JSON to define a GrafanaDashboard CR (Custom Resource). If the JSON is invalid, the dashboard will not appear in Grafana web UI. Below is a sample GrafanaDashboard CR YAML.
 
-    ```yaml
-    apiVersion: integreatly.org/v1alpha1
-    kind: GrafanaDashboard
-    metadata:
-      name: grafana-dashboard-example
-      namespace: test-ns
-      labels:
-        grafanaDashboard: grafana-operator
-    spec:
-      json: |-
+```yaml
```

Comment:
I think the indentation needs to be kept as is, otherwise the formatting is incorrect, this is within a numbered list, why did you change it like this?

Reply:
We can either add yaml by enclosing it in ```yaml ... ``` or by just adding 4 spaces. 
Here both of the methods were applied i.e. adding ```yaml and spaces as well due to which the yaml was invalid and tag (```yaml) was displaying. Removing spaces resolved the issue. 

You can let me know whatever looks right, I can use that method.
1. With indentation (I have added 6 spaces - 4 to add it as yaml and 2 extra spaces for indentation):
![Screenshot from 2024-07-18 14-17-40](https://github.com/user-attachments/assets/c43192bc-2481-4568-be26-c83e9725f589)


2. With ```yaml tag
![Screenshot from 2024-07-18 14-20-47](https://github.com/user-attachments/assets/48c54de4-59ae-4f2e-806c-f718ee87b7d5)

Reply:
And in the first method, we can add more spaces as well

Reply:
Resolved, looks like this now:
![Screenshot from 2024-07-18 14-31-28](https://github.com/user-attachments/assets/38f7d154-9ce5-4d3e-9936-09fe7b5182b1)

Reply:
It's step 2 and so on that you need to make sure really renders

Reply:
corrected indentation, it renders correctly now. Screenshots:
![Screenshot from 2024-07-19 18-25-15](https://github.com/user-attachments/assets/68404b7b-7a03-4e1b-9918-7ea3352567bb)
![Screenshot from 2024-07-19 18-25-56](https://github.com/user-attachments/assets/5acdad21-9f47-44a7-a338-a254678b44ea)